README.md

```diff
@@ -1 +1 @@
-test/README.lhs
```

Comment:
Oh, I totally missed that this was a symlink, just to a not-top-level file. I will re-un-correct this.